docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -153,7 +153,7 @@ Using the  <TechnicalTag relative="../../../" tag="data_context" text="Data Cont
 In the example, we have added a Datasource that connects to data in GCS using a Pandas dataframe. The name of the new datasource is `gcs_datasource` and it refers to a GCS bucket named `test_docs_data`.
 
 For more details on how to configure the Datasource, and additional information on authentication, please refer to [How to connect to data on GCS using Pandas
-](../guides/connecting_to_your_data/cloud/gcs/pandas.md)
+](/docs/0.15.50/guides/connecting_to_your_data/cloud/gcs/pandas)
```

Comment:
we have a new doc for this, why are you linking to v15?

Reply:
we can link to this doc instead https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/fluent/cloud/how_to_connect_to_data_on_gcs_using_pandas/